tools/dependency/release_dates.py

```diff
@@ -111,6 +117,51 @@ def issues_exist(title, git):
     return issues.totalCount > 0
 
 
+# search for issue by title and delete old issue if new package version is available
+def search_old_version_open_issue_exist(title, git, package_repo, latest_release):
+    # search for only "Newer release available `{dep}`:" as will be common in dep issue
+    title_search = title[0:title.index(":")]
+    query = f'repo:envoyproxy/envoy {title_search} in:title is:open'
```

Comment:
can we shift the `envoyproxy/envoy` part to a constant

the reason for asking this is that im testing against my own fork, and currently i have to go through and edit every instance

(also im thinking to move this to an object-oriented pattern fairly soon, and will likely use the constant as a property of the class)

Reply:
ye its should be moved to a constant. Let me refactor it